src/app/pages/apps/store/available-apps-store.service.ts

```diff
@@ -393,4 +361,120 @@ export class AvailableAppsStore extends ComponentStore<AvailableAppsState> {
       }),
     );
   }
+
+  private sortAppsByName(filteredApps: AvailableApp[]): AppsByCategory[] {
+    const appsByCategory: AppsByCategory[] = [];
+
+    const alphabet = [
```

Comment:
Not really sustainable. There may be an app starting with a number or cyrillic letter. Need to look at existing letters in catalog apps.

Reply:
@undsoft - you mean check all the apps names and get unique first letters?

Reply:
Yup.